contracts/src/v0.8/llo-feeds/VerifierProxy.sol

```diff
@@ -121,24 +121,30 @@ contract VerifierProxy is IVerifierProxy, ConfirmedOwner, TypeAndVersionInterfac
   }
 
   /// @inheritdoc IVerifierProxy
-  function verify(bytes calldata payload) external payable checkAccess returns (bytes memory) {
+  function verify(
+    bytes calldata payload,
+    bytes calldata feePayload
```

Comment:
Is this always going to be feePayload? or should we change this to parameterPayload - to save potential future confusion

Reply:
Yeah I agree with that. I'll change the naming

Reply:
Would it make sense to forward this to the `Verifier` as well?

Reply:
Updated to `parameterPayload` everywhere for clarity

Reply:
Keeping the parameterPayload out of the Verifier, since each `Verifier.verify` request has exactly one report which can embed any verification-specific payload.